operations/jsonnet/microservices/configmap.libsonnet

```diff
@@ -8,6 +8,11 @@
     server: {
       http_listen_port: $._config.port,
     },
+    live_store: {
+      partition_ring: {
+        delete_inactive_partition_after: $._config.live_store.downscale_delay,
+      },
+    },
```

Comment:
Can you change this to `tempo_live_store_config` (line 112) so the blast radius is smaller? Otherwise this is propagated to all components